packages/website-eslint/rollup.config.js

```diff
@@ -108,6 +116,16 @@ module.exports = {
           test: /process\.env\.IGNORE_TEST_WIN32/u,
           replace: 'true',
         },
+        {
+          // replace all process.env.IGNORE_TEST_WIN32 with true
```

Comment:
```suggestion
          // replace all process.cwd() with true
```

Though: does the `()` need to be escaped?